passive/s3.js

```diff
@@ -0,0 +1,40 @@
+// S3 bucket finder by alishasinghania09@gmail.com 
+
+function scan(ps, msg, src) {
+    // populate some parameters which will be needed if s3 bucket url is present
+    var alertRisk = 1
+    var alertReliability = 4
+    var alertTitle = 's3 bucket url'
+    var alertDesc = 's3 bucket url found in response'
+    var alertSolution = 'Remove s3 buckets name from response or make sure the permissions in bucket are configured properly'
+    var cweId = 0
+    var wascId = 0
```

Comment:
```suggestion
    var cweId = 200
    var wascId = 13
```

Information leakage: http://projects.webappsec.org/w/page/13246975/Threat%20Classification%20Taxonomy%20Cross%20Reference%20View

Reply:
Made the suggested changes in https://github.com/zaproxy/community-scripts/pull/213/commits/8b9fcfbef900d1c852e6c98de6fffce7ec5bf6eb commit